attr/value.go

```diff
@@ -13,11 +13,16 @@ type Value interface {
 	// Type returns the Type that created the Value.
 	Type(context.Context) Type
 
-	// ToTerraformValue returns the data contained in the Value as
-	// a tftypes.Value.
+	// ToTerraformValue returns the data contained in the Value as a tftypes.Value.
 	ToTerraformValue(context.Context) (tftypes.Value, error)
 
 	// Equal must return true if the Value is considered semantically equal
 	// to the Value passed as an argument.
 	Equal(Value) bool
+
+	// Unknown returns true if the Value is not known.
+	Unknown() bool
```

Comment:
I'd recommend naming these with an `Is` prefix, so the existing fields can remain (at least for now).

Reply:
OK.

But follow up: in regards to `IsFullyKnown` and `IsKnown` on TF types, they use the _positive_ connotation. While the FW types use the negative `Unknown`.

While it's of course trivial to obtain the opposite for `Unknown / Known`, it's not the same for `FullyKnown / FullyUnknown`? Because something that is _not_ fully known, can be fully or partially unknown.

I wouldn't care at all about `FullyKnown`-ness, but we do have the type `Object` to deal with. What's your take?

Reply:
We can add fully known handling later, if it is necessary. 😄  For now, I think its decent to only worry about directly checking a specific value.

Reply:
Even by adding it later, I think it's important to decide what we do about `Unknown vs Known`. Because if we add it later and we stick for now with `Unknown`, we would end up with `Unknown()` and `FullyKnown()`? Or with `Unknown()` and `FullyUnknown()`?

I appreciate that this might seem superfluous right now, but around future-proofing naming I get jittery.